docs/assets/templates/tools/.dockstore.yml

```diff
@@ -0,0 +1,60 @@
+# The first line refers to the version 1.2 of the .dockstore.yml schema
+version: 1.2
+
+# An array of tools. Each element corresponds to a tool entry on Dockstore.
+tools:
+  
+  # The optional name for a tool, which may only consist of alphanumerics
+  # and internal underscores and hyphens. Names may not exceed 256 characters. 
+  # If using a ``/.dockstore.yml`` with multiple tools, this field is required 
+  # to uniquely identify tools in the repository. Otherwise, it is optional.
```

Comment:
@NatalieEO Is this true? I did not know that.

Reply:
Should be, this is carried over from workflows.

Reply:
yes, it's carry over like Denis mentioned